dlpy/applications.py

```diff
@@ -383,6 +389,7 @@ def VGG13(conn, model_table='VGG13', n_classes=1000, n_channels=3, width=224, he
         Specifies the CAS connection object.
     model_table : string, optional
         Specifies the name of CAS table to store the model.
+        Default: VGG13
```

Comment:
for model_table, it is just a name, we don't need to explicitly indicate its default value. could you please remove this and all similar ones from your push. thanks